scripts/variantstore/wdl/GvsCreateVAT.wdl

```diff
@@ -165,10 +138,14 @@ task MakeSubpopulationFiles {
     # ------------------------------------------------
     # Outputs:
     output {
+        File vat_schema_json_file = vat_schema_json_filename
+        File variant_transcript_schema_json_file = variant_transcript_schema_json_filename
+        File genes_schema_json_file = genes_schema_json_filename
+
         File ancestry_mapping_list = output_ancestry_filename
         File custom_annotations_template_file = custom_annotations_template_filename
-        Array[File] input_vcfs = read_lines(updated_input_vcfs_file)
-        Array[File] input_vcf_indices = read_lines(updated_input_indices_file)
+        Array[File] input_vcfs = read_lines(inputFileofFileNames)
+        Array[File] input_vcf_indices = read_lines(inputFileofIndexFileNames)
```

Comment:
preexisting issue but the mix of camel and snake cases is unfortunate

Reply:
Fixed!